server/src/middlewares/authentication.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require("jsonwebtoken");
+
+exports.createToken = function (id) {
+  const token = jwt.sign({ data: id }, process.env.JWT_SECRET, {
+    expiresIn: 60 * 60 * 12,
```

Comment:
Use a specific number instead of `60 * 60 * 12` or add a comment, so that others reading your code don't have to do the math or wonder what the result represents. Even better, `jsonwebtoken` allows more explicit strings like `40d` (40 days) or `36h` (36 hours)